scripts/binary_size_check.sh

```diff
@@ -1,8 +1,9 @@
-#!/bin/sh
-
+#!/bin/bash
```

Comment:
Most systems have sh in some capacity, whether the bin or a symlink. I did this for compatibility purposes, especially given the latest issue that came through about nonbash compatibility with dash.

Reply:
While I generally laud portability, all of our scripts are bash already, except manifest.sh which is likely an oversight.